### PR TITLE
ENH: Reader for Park Systems TIFF

### PR DESCRIPTION
**Pull request by pastewka**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -153,6 +153,8 @@ def run(self):
         'python-dateutil',
     ],
     extras_require={
-        'mitutoyo': []
```
Comment (jotelha) [inline setup.py:156]: Hypothetically, you want to keep those empty extras once they had been introduced.

**Review by jotelha — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (pastewka) [reply setup.py:156]: What do you need them for?

Comment (pastewka) [reply setup.py:156]: Ah, once they had been introduced in a prior version such that install SurfaceTopography [mitutoyo] continues to work? Why is the openpyxl dependency not in there?

**Comment by pastewka** [comment]
You mean the 'tifffile' dependency?

**Comment by pastewka** [comment]
I can also wrap this into the default dependencies if this causes trouble

**Comment by jotelha** [comment]
Yes, it will break something somewehere in the automized discovery and import of Readers in the tests when the Reader tries to import a module that's not available, and that will only show in the tests of packages that depend on SurfaceTopography, but do not install the extras (at least I saw this in the tests of SurfaceTopographyML for SurfaceTopography 1.1 where I introduced the mitutoyo extra). So if we want to use continue using "extras" to avoid bloating the core dependencies while adding input filters for exotic formats (which makes sense), we will need to introduce some kind of introspection in when building the list of available readers in https://github.com/ContactEngineering/SurfaceTopography/blob/d75ca1ebeb1e64b2e745939923ab7857c596f925/SurfaceTopography/IO/__init__.py#L57-L77

**Comment by pastewka** [comment]
I am not entirely sure how this is supposed to work. If I make the `tiffile` or `openpyxl` optional, then the PS or Mitutoyo readers stop working when I don't install these dependency. This means we will get an import error when trying to load anything, since all modules are loaded in `SurfaceTopography.IO` (for the format detection). Making those packages optional would mean some additional mechanism to disable the packages internally, which we don't have at present.

**Comment by jotelha** [comment]
Yes, we keep that in mind, for now maybe better just avoid extras? You see the effect here  now https://github.com/ContactEngineering/ContactMechanics/pull/84, the tests break because of that optional pandas dependency in 1.1.

**Pull request merged**